content/en/blog/2022-09-01-manage-kyverno-policies-as-ocirepositories/index.md

```diff
@@ -0,0 +1,213 @@
+---
+author: developer-guy
+date: 2022-09-01 11:30:00+00:00
```

Comment:
This means it'll only be available at 11:30 UTC today.

Reply:
that's probably ok.